### ISSUE TITLE
Can't install go-sdl2 on Win10

### ISSUE THREAD

**Issue by Diamondtroller** [veandco/go-sdl2#473]
I can't install sdl package because it returns this error:
```
# github.com/veandco/go-sdl2/sdl
In file included from go\src\github.com\veandco\go-sdl2\sdl\audio.go:4:
./sdl_wrapper.h:2:11: fatal error: SDL2/SDL.h: No such file or directory
  #include <SDL2/SDL.h>
           ^~~~~~~~~~~~
compilation terminated.
```

I'm using git bash: GNU bash, version 4.4.23(1)-release (x86_64-pc-msys)
Go: go version go1.15.2 windows/amd64

**Comment by veeableful** [comment]
Hi @Diamondtroller, have you installed the native SDL2 libraries? You can follow the steps on the README.

**Comment by Diamondtroller** [comment]
Sorry @veeableful I didn't install libs correctly, after I posted this issue I only then checked closed issues and saw #472. This was my first issue in github, sorry for inconvenience.
